bears/yml/RAMLLintBear.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='ramllint',
+        use_stdout=False,
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(?P<severity>error) (?P<filename>.+)\n  (?P<message>.+)')
```

Comment:
You don't have to catch the filename, and you aren't catching the name of the rule (origin). 
The regex should look something like `(?P<severity>error).*\n   (?P<message>.+) \[(?P<origin>.+)\]` assuming `POST http://example.com/resource` isn't a part of the message.